readme.md

```diff
@@ -29,6 +29,8 @@ Everyone
 - Have two matrices, one for min_x and one for energies so that we can use typed arrays
 - Could convert picture rgba array to [`Uint32Array` rgb number array](https://hacks.mozilla.org/2011/12/faster-canvas-pixel-manipulation-with-typed-arrays/) to save space
 - Do logical deletes on the energy matrix. This way the cost of deletion goes way down, with the cost of finding the neighbor when recalculating going up some.
+- Only recalculate energy of pixels right beside the removed seam. That is, only for those that removing the seam could change the energy. The cumulative sum unfortunately does have to be recalculated for almost every pixel.
```

Comment:
Good idea! Doesn't change the worst case scenario, but might make things better ...